infrastructure/prod.tfvars

```diff
@@ -4,3 +4,4 @@ api_gateway_test_valid_certificate_thumbprint = "3B0D23863EE7EF90290AD02B0ACFD7C
 allowed_client_certificate_thumbprints = ["919F8A770C2A2F06933BA11551001ED0581B8848"]
 
 blob_processing_delay_in_minutes = "30"
+error_notifications_enabled = "false"
```

Comment:
I'm a fan of default disabling and enabling explicitly for each environment, so that nothing is enabled in prod by mistake. This is just a preference, so up to you.

Reply:
depends on a feature, i suppose. right now error notifications were developed "enabled" by default. so it make no sense to disable consumption by default. except those environments which matter.

i have no preference just try to make approach sensible. all the message should be dead lettered by now so even if enabled now on some testing environment - it'll be fine to test out the functionality as discussed few times.

p.s. there should not be a mistake on production. it is disabled. in order to enable there has to be manual intervention

p.p.s. as mentioned in description - can disable globally if requested. stating as preference is not an argument giving the position described above ^^